### PR TITLE
feat: bump salt config version

### PR DESCRIPTION
**Pull request by haveiss**
### Context
- Relates with https://github.com/odpf/salt/pull/66
- To fix telemetry headers configuration via environment variable

### CODE REVIEW

**Review by rahmatrhd — APPROVED** [review]
should we wait until the salt PR is merged into main and then bump to that version?

**Comment by haveiss** [comment]
> should we wait until the salt PR is merged into main and then bump to that version?

latest salt version are breaking changes https://github.com/odpf/salt/pull/63, it requires to make changes, let's do that on separate PR, wdyt @rahmatrhd ?

**Pull request merged**
